.github/workflows/unit-tests.yml

```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 jobs:
   container-test-job:
     runs-on: ubuntu-latest
-    container: maierbn/opendihu:latest
+    container: carmehp/opendihu:latest
```

Comment:
To not end up in the same situation as now in a few years, one could also think of setting up an `opendihu` account.

Reply:
For this, a simple user account should be fine. The important part is the handle (`opendihu`), not the email: you can change the email later.

But you can also ignore this for now and only look into it in a year or two. In any case, even if you continue with `carmehp/opendihu`, it will not be difficult for someone else to do the migration in the future.

Reply:
I agree, this is not good for the long term. For now I leave it like this